lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+---
+name: Condition
+field_name_base_url: http://hl7.org/fhir/r4/condition-definitions.html#Condition
+fields:
+- name: clinicalStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: active | inactive | resolved
+  url: http://hl7.org/fhir/r4/condition-definitions.html#Condition.clinicalStatus
+  binding:
+    description: The clinical status of the condition.
+    terminology:
+      - display: Condition Clinical Status Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-clinical
+        info_link: http://hl7.org/fhir/r4/valueset-condition-clinical.html
+
+- name: verificationStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: unconfirmed | confirmed | refuted | entered-in-error
```

Comment:
need to update to include all verification statuses.

Reply:
Added. [diff](https://github.com/cerner/fhir.cerner.com/pull/273/commits/4872fd105329788054750c0c23ab3e1c4ec4459a#diff-abb99d2139d79f8a2a8f1d214f5f6882R22)